examples/experimental/dagster-airlift/dagster_airlift/core/dag_defs.py

```diff
@@ -91,3 +91,30 @@ def task_defs(task_id, defs: Definitions) -> TaskDefs:
     by Airlift tooling.
     """
     return TaskDefs(task_id, defs)
+
+
+def standalone_dag_defs(
```

Comment:
this feels pretty good. I honestly don't mind having it be its own fxn

Reply:
Yeah. Makes the implementation simple which is great.